examples/lifelong_learning/atcii/README.md

```diff
@@ -16,15 +16,15 @@ In this example, you can use [ASHRAE Global Thermal Comfort Database II](https:/
 
 
 
-download [datasets](https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/atcii-classifier/dataset.tar.gz), including train, evaluation and incremental dataset.
+We provide a well-processed [datasets](https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/atcii-classifier/dataset.tar.gz), which including train, evaluation and incremental dataset.
 ```
```

Comment:
incremental =>retained ?  different from incremental job

Reply:
Lifelong Learning retrain should only uses icremental samples